cpp/cmake/cudf-config.cmake.in

```diff
@@ -51,11 +46,11 @@ set(_possible_targets_to_promote
       cudf::cudf
       cudf::benchmark
       cudf::benchmark_main
```

Comment:
Should we do the same for these as well?

Reply:
Done.